### DISCUSSION TITLE
When I add in a New Row through the UI NULL is inserted

### DISCUSSION THREAD

**Discussion by britestonedev** [Budibase/budibase#5116] [Help]
Hey Team,

I've connected an external datasource (MySQL) to budibase. I've fetched the tables and defined the relationships (I think). I have no data in the current Tables. I was trying to add in a row through the UI on my staff Table. 

The Column is setup as a VARCHAR(50) as it's just designed to hold staff names. When I add in a row through the UI it inserts NULL into the field. I'm hoping someone can tell me where I've gone wrong =/

![image](https://user-images.githubusercontent.com/102457267/160270446-3b8180a6-e204-4817-bad2-1a96d2d347bb.png)

![image](https://user-images.githubusercontent.com/102457267/160270458-2a6578df-304c-4fc2-8c3e-48539c857a42.png)
![image](https://user-images.githubusercontent.com/102457267/160270469-ab7ab73b-8eb8-4c99-9b5f-396cf5087b58.png)
![image](https://user-images.githubusercontent.com/102457267/160270479-38de0a88-c437-418b-a179-ab5779009528.png)

Any help would be appreciated, I tried to use BudiDB but it wasn't playing nice at all.

**Comment by melohagan** [comment]
Hey @britestonedev 

I had success adding a row from the UI with the following schema:

```
CREATE TABLE `staff` (
  `idstaff` int NOT NULL AUTO_INCREMENT,
  `staffName` varchar(50) DEFAULT NULL,
  PRIMARY KEY (`idstaff`),
  UNIQUE KEY `staffName_UNIQUE` (`staffName`)
)
```

What data types are those other fields in the table (ballcomp, tradetrans, etc)?

Whenever you click 'Create Row' can you see any error messages in the Network tab on the dev console? (Fn+F12 on Chrome).

**Reply by britestonedev** [reply]
Thanks for your response. I'll take a look.

But those additional columns are the ones generated from the relationships.
